frontend/src/components/MyProfile/MyProfile.vue

```diff
@@ -0,0 +1,104 @@
+<script setup>
+import DeleteIcon from '@/assets/img/delete_icon.svg';
+import ProfileIllustration from '@/assets/img/illustrations/profile_illustration.svg';
+import { useRouter } from 'vue-router';
+import userService from '@/api/users.js';
+import { useUserStore } from '@/stores/user';
+
+const userStore = useUserStore();
+const router = useRouter();
+const logout = () => {
+  userStore.logoutUser();
+  router.push({ name: 'auth' });
+};
+const deleteUser = async () => {
+  const isDeleteConfirmed = confirm('Do you really want to delete account?');
+  if (isDeleteConfirmed === false) return;
+
+  const deleteResponse = await userService.deleteUserProfile();
+  if ('error' in deleteResponse) return alert('Internal Server Error. User can not be deleted.'); // TODO: add better error handling
+
+  logout();
+};
+</script>
+
+<template>
+  <div class="main">
+    <div class="rsprd-bar">
+      <h2 class="rsprd-bar-title">My Profile</h2>
```

Comment:
if you are following BEM this class should be `rsprd-bar__title`
<img width="449" alt="image" src="https://user-images.githubusercontent.com/32748145/185627883-9aece0ab-7e4c-4426-a4ee-22b86be0f42e.png">
http://getbem.com/naming/

Reply:
Usually I don't use BEM but changed the code because of consistency since you use it.